tests/conftest.py

```diff
@@ -244,8 +311,16 @@ def test_session_tmp_path(test_fc_session_root_path):
 @pytest.fixture
 def results_file_dumper(request):
     """Yield the custom --dump-results-to-file test flag."""
+    generate_baselines = False
+    dump_results = False
+
     if request.config.getoption("--dump-results-to-file"):
-        return ResultsFileDumper(request.node.originalname)
+        dump_results = True
+    if request.config.getoption("--generate-baselines"):
```

Comment:
`getoption` has a `default` parameter that you can use:

`generate_baselines = request.config.getoption("--generate-baselines", default=False)`